src/import-data/ImportDataOverview.ts

```diff
@@ -33,12 +33,12 @@ const styles = {
 const getTitleForImportRequest = (importRequest: ImportRequest): string => {
   switch (importRequest.type) {
     case 'tdr-snapshot-export':
-      return `Importing snapshot ${importRequest.snapshot.name}`;
+      return `Import snapshot ${importRequest.snapshot.name}`;
```

Comment:
@bethsheets I went ahead and changed the tense of these other related titles from present continuous to simple present to match what's in the mockups; this wasn't explicitly called out in the Figma so I wanted to call attention to it here.